tests/integration/test_postgresql_replica_database_engine/test.py

```diff
@@ -979,6 +979,7 @@ def test_user_managed_slots(started_cluster):
     drop_postgres_table(cursor, table_name)
     drop_materialized_db()
     drop_replication_slot(replication_connection, slot_name)
+    cursor.execute(f'DROP TABLE IF EXISTS test_table')
```

Comment:
```suggestion
    cursor.execute(f'DROP TABLE IF EXISTS test_table')
```

BTW you don't need to use [formatted strings literals](https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings) here, since there are no expressions

Reply:
I know, this string looked like `f'DROP TABLE IF EXISTS {table_name}`, I removed `{table_name}` and forgot to remove `f`